app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -97,7 +97,7 @@
 
     private ContributionsListFragment contributionsListFragment;
     private static final String CONTRIBUTION_LIST_FRAGMENT_TAG = "ContributionListFragmentTag";
-    private MediaDetailPagerFragment mediaDetailPagerFragment;
+    static MediaDetailPagerFragment mediaDetailPagerFragment;
```

Comment:
I don't think this should be static - there should be a `MediaDetailPagerFragment` per `ContributionsFragment`